tests/performance/run_perf_tests.sh

```diff
@@ -29,15 +29,36 @@ exit 1
 }
 
 echo -n "Running performance test using MeTTa's built-in Atom Space..."
-log_file=$(mktemp --suffix='_perf_test_builtin').log
-time (metta ${PWD}/tests/performance/test_builtin.metta 2>&1>${log_file})
-echo "Check the log file for more details: ${log_file}"
+builtin_log_file=$(mktemp --suffix='_perf_test_builtin.log')
+time (metta ${PWD}/tests/performance/test_builtin.metta 2>&1>${builtin_log_file})
+echo "Check the log file for more details: ${builtin_log_file}"
 
 echo "-----"
 
 echo -n "Running performance test using DAS..."
-log_file=$(mktemp --suffix='_perf_test_das_ram_only').log
-time (metta ${PWD}/tests/performance/test_das_ram_only.metta 2>&1>${log_file})
-echo "Check the log file for more details: ${log_file}"
+das_log_file=$(mktemp --suffix='_perf_test_das_ram_only.log')
+time (metta ${PWD}/tests/performance/test_das_ram_only.metta 2>&1>${das_log_file})
+echo "Check the log file for more details: ${das_log_file}"
 
 echo "-----"
+
+echo -n "Comparing performance test results... "
```

Comment:
I'm not sure I understand what's been compared here. If this is a comparison between the script output in each case (built-in vs DAS Gate) so it's OK. But if this is comparing the execution time it doesn't make sense.

Reply:
Outputs are being compared to ensure that both tests are giving the same result.
I will change the messages to make this clear.